src/lib.rs

```diff
@@ -400,4 +405,12 @@ mod rut_test {
             Error::OutOfRange.to_string()
         );
     }
+
+    #[test]
+    fn support_rut_with_preceding_zeroes() {
+        let have = Rut::from("0017951585-7").unwrap();
+        let expect = Rut::from("17951585-7").unwrap();
+
+        assert_eq!(have, expect);
+    }
```

Comment:
I started with the example provided by @jfgodoy here but got a `InvalidFormat` error, so I reused one of the RUT numbers in above tests